.evergreen/config.yml

```diff
@@ -1900,7 +1900,7 @@ axes:
         display_name: "Ubuntu 14.04"
         run_on: ubuntu1404-test
         variables:
-          GO_DIST: "/opt/golang/go1.16"
+          GO_DIST: "/opt/golang/go1.17"
           # The GCC toolchain that comes with Ubuntu 14.04 is incompatible with the standard binary
           # release of Go 1.16. As a result, Cgo builds on Ubuntu 14.04 fail with a linking error
```

Comment:
Is this still true of Go 1.17? (I bet it is, but we'll just want to update this comment).

Reply:
I would assume so, but I can test by removing the `GO_BUILD_TAGS` override to see if it fails. If it still fails, I'll update the comment to say "Go 1.16+".

Reply:
The Ubuntu 14.04 tasks do seem to pass with that line commented out; so you should be able to just remove this comment and the `GO_BUILD_TAGS` 🤔 .

Reply:
Yeah, I thought that was a bit odd so I looked into it further and realized Go 1.17 isn't available on the Ubuntu 14.04 image, so Go test was running with Go 1.7. It's useful to note that there are other Go installations in the `PATH`, so it's easy to unintentionally get the wrong Go version if something is messed up or missing.

As far as a fix, I've just reverted the Ubuntu 14.04 build to use Go 1.16, added a comment about why, and left the overridden `GO_BUILD_TAGS`.

Reply:
Sounds good!